public String divide(Integer a, Integer b){
    try{
    Integer C = a / b;
    return  string.valueOf(C);
    }
    catch(Exception e) {
    System.debug('The following exception has occurred: ' + e.getMessage());
}
}